### ISSUE TITLE
Virtual Sensor: "Your Current Location" only once....

### ISSUE THREAD

**Issue by DG12** [ruuvi/com.ruuvi.station.ios#29]
1) Once a virtual sensor for  "Your Current Location" is set , adding another should be disabled.

2) Maybe it would be good to come up with a better default name since the display shows:
"YOUR CURRENT LOCATI..." Perhaps simply "Our Location" or "Your Location"
2a) I still don't like converting the name (in mixed case) to all CAPS.

3) Perhaps the explanation in the documentation could include something like "which follows you". I interpreted it to mean where I was at the the time. That is, a static location based on my live current location. (but maybe it's just me).

4) The humidity and hPa MUST be displayed in integer percent since OpenWeatherMap does not return fractional values.

5) Wind speed (configurable in Miles/hour or meters/second or implicit since fahrenheit user probably wants mph) would be a nice addition since it's available.

**Comment by rinat-enikeev** [comment]
@DG12 for 1, 2, 4 please check 0.4(37). 

I don't think changing UI for 5 is worth it.

**Comment by rinat-enikeev** [comment]
@DG12 can we close this?

**Comment by DG12** [comment]
Looking good.

Regarding wind speed and additional thoughts. 

For me the purpose of observing  the current environmental conditions  is usually to answer the question:" What should I wear?" 
After looking at the temperature(ignoring the noise of hundredths of a degree) and the humidity I don't actually care about the values but rather I try to interpret them into categories like cold and damp, hot and damp, warm and not-damp etc.

Wind speed at ground level, although very local, contributes to that evaluation.

I previously came up with the idea of mounting a ruuvi from a vertical, swingable rod on a tree limb and using the accelerometer to determine wind "speed". Again looking at values like CALM, LIGHT, BREEZY and STRONG. At that time MovementCounter was not yet implemented. Now that it is I will be mounting one on the "weather arm" I have mounted on a tree. I'll be watching the change in movementCounter over time

Incident sun light is also a very important consideration.I am also planning on adding a logarithmic light sensor inside a clear case to the one swinging.

A second ruuvi one I plan on adding will be attached to a tip-cup rain gage, again using the movementCounter to count the cycling and measure rainfall.

Finally I am unable to use air pressure in any weather consideration. I know that the  direction of change (i.e. sign of first derivative ) is supposed to indicate a change in the general weather for example stormy . For that I proposed a format 6 ( current 5 plus 1 byte for the change in pressure) but nobody seemed to be interested in that. I have not been able to actually confirm and correlation  I appreciate any thought you had on that.
